src/intervals/rounding.jl

```diff
@@ -9,6 +9,7 @@ interval rounding types, e.g.
 +(IntervalRounding{:none}, a, b, RoundDown)
 
 The current allowed rounding types are
+- :errorfree # use errorfree arithmetic via FastRounding.jl
```

Comment:
Following the standard, how about `:errorfree` -> `:fast`; `:fast` -> `:accurate`; `:correct`-> `:tight`?